autoload/go/cmd.vim

```diff
@@ -117,16 +117,15 @@ endfunction
 
 " Run runs the current file (and their dependencies if any) in a new terminal.
 function! go#cmd#RunTerm(bang, mode, files) abort
-  let cmd = "go run "
-  let tags = go#config#BuildTags()
-  if len(tags) > 0
-    let cmd .= "-tags " . go#util#Shellescape(tags) . " "
+  let cmd = ["go", "run"]
```

Comment:
@bhcleek just fixed this at work today - build tags weren't working correctly. looks ok? needed to len the resulting string. The results of #BuildTag() will always be a string correct?

Reply:
yeah, this looks right now.